Detections/AWSCloudTrail/AWS_ChangeToVPC.yaml

```diff
@@ -1,15 +1,17 @@
 ﻿id: 65360bb0-8986-4ade-a89d-af3cf44d28aa
 name: Changes to Amazon VPC settings
 description: |
-  'This alert monitors changes to Amazon VPC (Virtual Private Cloud) settings such as new ACL entries and routes in route tables.
+  'Amazon Virtual Private Cloud (Amazon VPC) lets you provision a logically isolated section of the AWS Cloud where you can launch AWS resources
+  in a virtual network that you define.
+  This alert monitors changes to Amazon VPC (Virtual Private Cloud) settings such as new ACL entries,routes, routetable or Gateways.
   More information: https://medium.com/@GorillaStack/the-most-important-aws-cloudtrail-security-events-to-track-a5b9873f8255 
-  and https://aws.amazon.com/vpc/.'
+  and AWS VPC API Docs: https://docs.aws.amazon.com/AWSEC2/latest/APIReference/OperationList-query-vpc.html'
 severity: Low
 requiredDataConnectors:
  - connectorId: AWS
    dataTypes:
     - AWSCloudTrail
-queryFrequency: 1d
+queryFrequency: 1h
```

Comment:
Again, lets be sure on the 1h.  This needs to only be done if it is a High severity detection.